runtime/nsd_ossl.c

```diff
@@ -85,6 +85,11 @@ static int bGlblSrvrInitDone = 0;	/**< 0 - server global init not yet done, 1 -
 /* Main OpenSSL CTX pointer */
 static SSL_CTX *ctx;
 
+/* Static Helper variables for CERT status */
+short bHaveCA;
+short bHaveCert;
+short bHaveKey;
```

Comment:
the comment says these are meant as `static`...

Reply:
correct they should be static